ext/pango/gstbasetextoverlay.c

```diff
@@ -2080,16 +2080,21 @@ gint x0, gint x1, gint y0, gint y1) \
 { \
   gint i, j;\
   guint8 *dest_ptr;\
+  gint alpha_offset[2] = { 3, 0 };\
```

Comment:
The compiler can probably optimise this out, but consider making this a const by using a ternary, instead of dereferencing an array in a deeply nested loop.

Reply:
probably clearer that way too, will do